apps/web/src/providers/modal-provider.tsx

```diff
@@ -152,7 +152,7 @@ export function Modals() {
               setOpenCreateWorkspaceModal(false);
               setOpenWorkspacesModal(false);
               setCurrentWorkspace(id);
-              return jumpToSubPath(id, WorkspaceSubPath.SETTING);
+              return jumpToSubPath(id, WorkspaceSubPath.ALL);
```

Comment:
I think this is a requirement for creating new workspace that we will redirect to the settings pages after creation.